assets/src/edit-story/app/story/useStoryReducer/reducers/addPage.js

```diff
@@ -65,7 +65,7 @@ function addPage(state, { page, position }) {
       ...state.pages.slice(insertionPoint),
     ],
     current: id,
-    selection: [],
+    selection: [page.elements[0].id],
```

Comment:
Added this here so that a newly added page will always get selected. Is there a case where this should not happen?

Reply:
When you change the current page, nothing is selected, so shouldn't it be the same when you add a new page?

Reply:
When adding an element (or a template now) then the added thing will get selected though. Perhaps the page should get selected when changing the current page, too? Why would it not?

Reply:
Hm, good question. Let's leave it as is, but perhaps file a follow-ticket with **UX Needed** to resolve it.